README.md

```diff
@@ -1,7 +1,6 @@
 <h1 align="center">Inlyne - a GPU powered, browserless, markdown + html viewer</h1>
 
 <p align="center">
-<a href="https://crates.io/crates/inlyne" target="_blank"><img height=25 src="https://img.shields.io/crates/v/inlyne.svg" /></a>
```

Comment:
Was this supposed to be removed?

Reply:
Yes certainly not suppose to be removed oops. Was just testing if the presence of images induced any blocking. I'll fix this up thanks.